.circleci/config.yml

```diff
@@ -79,34 +79,30 @@ jobs:
                 bename: gem_install
                 becommand: sudo gem install fpm
             - run: $GOPATH/bin/honeytail-linux-amd64 --write_default_config > ./honeytail.conf
+            - run: mkdir -p ~/artifacts
             - buildevents/berun:
                 bename: build_deb_amd64
-                becommand: ./build-pkg.sh -m amd64 -v "${CIRCLE_TAG}" -t deb
+                becommand: |
+                  ./build-pkg.sh -m amd64 -v "${CIRCLE_TAG}" -t deb
+                  mv *.deb ~/artifacts
             - buildevents/berun:
                 bename: build_deb_arm64
-                becommand: ./build-pkg.sh -m arm64 -v "${CIRCLE_TAG}" -t deb
+                becommand: |
+                  ./build-pkg.sh -m arm64 -v "${CIRCLE_TAG}" -t deb
+                  mv *.deb ~/artifacts
```

Comment:
I don't think `becommand` can take multiple commands in the same way the regular `run` command does - I think `becommand` is restricted to a single thing executable by a shell. If this actually works then my memory is wrong and carry on!

Reply:
Looking at the build output, I think the result of this syntax is that it runs the `./build-pkg.sh` within the scope of the span representing `build_deb_arm64` but the execution of the `mv` command is outside of it. So while both commands might both be executed, only the former is represented by the span. This makes the code here misleading in terms of what the trace output will represent. :/ Not quite as bad as I feared (that it wouldn't run at all) but we shouldn't land it like this.

Reply:
Cool, I will fix it. I'd note that we do it this way in at least one other repo: https://github.com/honeycombio/basenji/blob/master/.circleci/config.yml
Do you think it's possible to fix `becommand` to handle this case properly?